geospaas_harvesting/crawlers.py

```diff
@@ -230,15 +235,31 @@ def _get_links(cls, html):
         parser.feed(html)
         return parser.links
 
+class OpenDAPCrawler(BaseCrawler):
+    LOGGER = logging.getLogger(__name__ + '.OpenDAPCrawler')
+    FOLDERS_SUFFIXES = ('/contents.html')
+    FILES_SUFFIXES = ('.nc', '.nc.gz')
+    EXCLUDE = ('?')
+    def url_exact_navigation(self, resource_url):
+        return resource_url
+
+class ThreddsCrawler(BaseCrawler):
+    LOGGER = logging.getLogger(__name__ + '.ThreddsCrawler')
+    FOLDERS_SUFFIXES = ('/catalog.html',)
+    FILES_SUFFIXES = ('.nc', '.nc.gz')
+    EXCLUDE = ['/thredds/','http','_sh_polstere','ease',] #We exclude "EASE-Grid map projections" and "southern hemispheres" from harvesting provess
+    def url_exact_navigation(self, resource_url):
+        return resource_url.replace(resource_url[resource_url.find("catalog/"):resource_url.find("?dataset=")+9],"dodsC/")+'.dods'
+
 
-class CopernicusOpenSearchAPICrawler(Crawler):
+class CopernicusOpenSearchAPICrawler(BaseCrawler):
```

Comment:
As mentioned in another comment, BaseCrawler is not the parent class for all crawlers, and the `CopernicusOpenSearchAPICrawler` **should not** inherit from it. The way to access the target repository is completely different.
```suggestion
class CopernicusOpenSearchAPICrawler(Crawler):
```

Reply:
Done in the next commit.